### ISSUE TITLE
allow user define a custom value if the file doesn't exist or is empty

### ISSUE THREAD

**Issue by asuper0** [rust-cli/confy#97]
```rust
pub fn load_path_or_else<T, F>(
    path: impl AsRef<Path>,
    op: F
) -> Result<T, ConfyError>
where
    T: DeserializeOwned + Serialize,
    F: FnOnce() -> T,
{ }
```

Add the function so that user can pass a closure, which could return a custom value if the file doesn't exist or is empty.

**Comment by asuper0** [comment]
could solve #85

**Comment by deg4uss3r** [comment]
closed by #98.
